### PR TITLE
feat(asi-117): add missing appellant submission command fields

### PR DESCRIPTION
**Pull request by Si3rr4wow**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js [hunk 1]
```diff
@@ -146,7 +150,9 @@ exports.formatApplicationSubmissionUsers = ({
 			firstName: appellantFirstName,
 			lastName: appellantLastName,
 			emailAddress: null,
-			serviceUserType: 'Appellant'
+			serviceUserType: 'Appellant',
+			telephoneNumber: appellantPhoneNumber,
```
Comment (bryn-pins) [inline packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js:154]: I think this field is named wrong in submission, should be contactPhoneNumber, it's asked to the person filling out the appeal, not asking for appellant

Can we rename it in submission table, fieldName in questions and then add to just the primary contact

Comment (Si3rr4wow) [reply packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js:154]: Yeah agreed, this did feel like weird handling

File: packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js [hunk 2]
```diff
@@ -137,7 +139,9 @@ exports.formatApplicationSubmissionUsers = ({
 			firstName: contactFirstName,
 			lastName: contactLastName,
 			emailAddress: email,
-			serviceUserType: isAppellant ? 'Appellant' : 'Agent'
+			serviceUserType: isAppellant ? 'Appellant' : 'Agent',
+			telephoneNumber: isAppellant ? contactPhoneNumber : null,
+			organisation: isAppellant ? contactCompanyName : null
 		}
 	];
 	if (!isAppellant) {
```
[the remark this answers is not in the document]
Comment (Si3rr4wow) [reply packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js:148]: Now that there's only the one turnery in there for serviceUserType I don't think it's that bad. Take a look and lemme know if you still want me to make this change

[the remark this answers is not in the document]
Comment (bryn-pins) [reply packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js:148]: happy to leave
integration test has failed though:
'Formats HAS appeal submission then sends it to back office'

**Pull request merged**